app/src/main/java/com/example/bigowlapp/activityPage/MainPageActivity.java

```diff
@@ -7,13 +7,13 @@
 
 import com.example.bigowlapp.R;
 
-public class MainActivity extends AppCompatActivity {
+public class MainPageActivity extends AppCompatActivity {
```

Comment:
I think MainActivity is simply used to indicate the first activity that launches on app run, so it's not really a page. I almost think of it like a Java main function, but I'm not sure if that is good practice or not and would want others and your opinion.

Reply:
Agreed.

Reply:
I changed it to MainActivity. I think it is true to leave it as it is.